### PR TITLE
Introduces the ``calibrationlamp`` instrument. 

### PR DESCRIPTION
**Pull request by tribeiro**
As suggested on issue #83. 

* Removed all references to ``domelights``.
* Added interface, base instrument and fake instrument.
* Modified chimera-dome script for the new usage. It is possible to work with more than one lamp easily. When info is issued it will show the status of each lamp (or nothing, if none is present).

To configure a single lamp:
```
  calibrationlamp:
    - name: fake
      type: FakeCalibrationLamp
      dome_az: 0
      telescope_alt: 90
      telescope_az: 45
```

On `chimera-dome --info`:

```
======================================== 
Dome: /FakeDome/fake (/dev/ttyS1). 
Current dome azimuth: 42.13. 
Current dome mode: None. 
Dome slit is closed. 
======================================== 
Lamp /FakeCalibrationLamp/fake: OFF 
======================================== 
```

Or a series of lamps:

```
  calibrationlamps:
    - name: fake1
      type: FakeCalibrationLamp
      dome_az: 0
      telescope_alt: 90
      telescope_az: 45

    - name: fake2
      type: FakeCalibrationLamp
      dome_az: 10
      telescope_alt: 90
      telescope_az: 55
```

On `chimera-dome --info`:

```
======================================== 
Dome: /FakeDome/fake (/dev/ttyS1). 
Current dome azimuth: 42.13. 
Current dome mode: None. 
Dome slit is closed. 
======================================== 
Lamp /FakeCalibrationLamp/fake1: OFF 
Lamp /FakeCalibrationLamp/fake2: OFF 
======================================== 
```

### CODE REVIEW

File: src/scripts/chimera-dome [hunk 1]
```diff
@@ -46,6 +48,18 @@ class ChimeraDome (ChimeraCLI):
...
+        self.addParameters(dict(name="calibrationlamp",
+                                default="/CalibrationLamp/0",
+                                helpGroup="CALIBRATIONLAMP",
+                                help="Calibration lamp to be used."))
+
+        self.addParameters(dict(name="intensity",
+                                default=None,
+                                type=float,
+                                helpGroup="CALIBRATIONLAMP",
+                                help="Select calibration lamp intensity (from 0 to 100.)."))
```
Comment (wschoenell) [inline src/scripts/chimera-dome]: Intensity can be something different of this interval in lamps other than the fake. I think that this should come from the configuration like on the focuser.
https://github.com/astroufsc/chimera/blob/7d38c7e098c00de9c96ba9ba450f27de1985cceb/src/chimera/instruments/fakefocuser.py#L77-L78

**Comment by wschoenell** [comment]
These two help groups of `chimera-dome` could be merged....
```

  Dome lights control:
    --lights-off        Turn flat light off
    --lights-on         Turn flat light on

  CalibrationLamp:
    --intensity=INTENSITY
                        Select calibration lamp intensity (from 0 to 100.).
    --calibrationlamp=CALIBRATIONLAMP
                        Calibration lamp to be used.
                        [default=/CalibrationLamp/0]
```

**Comment by tribeiro** [comment]
Done...

**Comment by tribeiro** [comment]
And done...

**Comment by wschoenell** [comment]
Please add a configuration example to the fake instrument here:
https://github.com/astroufsc/chimera/blob/37c8bd88f66f3f8e59e569f92ad09e6cdb8eeb9a/src/chimera/core/chimera.sample.config

**Comment by tribeiro** [comment]
Done

**Pull request merged**